h/templates/accounts/base.html.jinja2

```diff
@@ -0,0 +1,33 @@
+{% extends "h:templates/layouts/base.html.jinja2" %}
+
+{% block header %}
+  {% include "h:templates/includes/logo-header.html.jinja2" %}
+{% endblock %}
+
+{% block content %}
+  <div class="form-container content">
+    {% include "h:templates/includes/flash-messages.html.jinja2" %}
+
+    <h1 class="form-header">
+      {% block page_title %}{% endblock %}
+    </h1>
+
+    <div class="form-description">
+      {% block form_message %}{% endblock %}
+    </div>
```

Comment:
You could if the `page_title` and `form_message` here to remove the empty divs.

Reply:
Done for `form_message`: https://github.com/hypothesis/h/pull/5964/commits/532f9bcdbfb921c205d071c5ff8c2ec663390aae I didn't bother with `page_title` 'cos that one is always set